docs/app/Examples/views/Card/Variations/Colored.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Card, Image } from 'stardust'
+
+const ColoredCards = () => (
+  <div>
+    <Card.Group itemsPerRow={4}>
+      <Card color='red'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='orange'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='yellow'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='olive'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='green'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='teal'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='blue'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='violet'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='purple'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='pink'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='brown'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='grey'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+    </Card.Group>
+  </div>
```

Comment:
This is correct, though we allow shorthand props _or_ subcomponent children to cover all cases.  There are essentially two APIs for every component, shorthand props and subcomponents.

The shorthand props API makes assumptions about where the the values go.  When a component takes an `icon` or `image` prop, it is placed as a sibling immediately before the `content`.  In many components, the presence of the icon/image is what dictates the use of the `content`.

If this use case does not meet the user's needs, then they can use the sub component API and place the Image (along with it's own props) anywhere they choose.

Please let me know if I've overlooked some other issue at play.